thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java

```diff
@@ -160,7 +156,12 @@ private DetectionConfigDTO buildDetectionConfigFromYaml(long tuningStartTime, lo
       tuningStartTime = tuningEndTime - TimeUnit.DAYS.toMillis(28);
     }
 
-    YamlDetectionConfigTranslator translator = this.translatorLoader.from(yamlConfig, this.provider);
+    YamlDetectionConfigTranslator translator;
+    try {
+      translator = this.translatorLoader.from(yamlConfig, this.provider);
+    } catch (Exception e) {
+      throw new IllegalArgumentException("Unable to instantiate the detection pipeline. Please verify the pipelineType");
```

Comment:
Can you pass the original exception out?

Reply:
Done!